### PR TITLE
Add slashing protection health check.

### LINKED ISSUE #130: Document slashing protection health check
See: https://github.com/ConsenSys/web3signer/pull/610

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: bgravenorst <byron.gravenorst@consensys.net>

## Describe the change

Add information about the slashing protection health check.

## Issue fixed

Fixes #130 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

## Preview
https://pegasys-web3signer--131.com.readthedocs.build/en/131/HowTo/Configure-Slashing-Protection/#health-checks

### CODE REVIEW

File: docs/HowTo/Configure-Slashing-Protection.md [hunk 1]
```diff
@@ -144,6 +144,17 @@ You can include additional optional pruning configuration options. For example,
 
     Do not use slashing protection database pruning while [importing or exporting the database](#import-or-export-a-slashing-protection-database).
 
+## Health checks
```
Comment (siladu) [inline docs/HowTo/Configure-Slashing-Protection.md:147]: @wcgcyx please correct me if I'm wrong but I _think_ the health check exists even when slashing protection is enabled, so might make sense to put this section somewhere more general. Potentially a new page under "Monitor nodes" alongside https://docs.web3signer.consensys.net/en/latest/HowTo/Monitor/Metrics/?

Comment (siladu) [reply docs/HowTo/Configure-Slashing-Protection.md:147]: It's also related to the upcheck: https://docs.web3signer.consensys.net/en/latest/HowTo/Get-Started/Start-Web3Signer/#confirm-web3signer-is-running

Comment (bgravenorst) [reply docs/HowTo/Configure-Slashing-Protection.md:147]: Ok, I renamed the section to "Slashing protection health check". I'll also add a note in the section regarding the upcheck.

**Review by rolandtyler — APPROVED** [review]
LGTM

**Pull request merged**